docs/docs/examples/workflow/citation_query_engine.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>Can we add more context about what this does? And not put CitationQueryEngine in the title but rather something like "Build RAG with in-line citations" </p><p> </p><p>We should link to the CitationQueryEngine cookbook in the docs for users to refer to if they want a "pre-built" implementation.</p><p> </p><p>Also can we add a Colab badge </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>